compiler/luci/service/src/CircleShapeInference.cpp

```diff
@@ -46,7 +46,11 @@ std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape
   {
     if (r)
       os << ",";
-    os << tensor_shape.dim(r).value();
+
+    if (tensor_shape.dim(r).known())
```

Comment:
(optional) is below possible?
```cxx
os << (tensor_shape.dim(r).known() ? tensor_shape.dim(r).value() : "?");
```

Reply:
I tried it but failed because `value()` and `"?"` is different type :(